benchmarks/benchmark_readers.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import random
+from pathlib import Path
+
+from aicsimageio import readers
+from aicsimageio.dimensions import DimensionNames
+
+###############################################################################
+
+# We only benchmark against local files as remote files are covered by unit tests
+# and are generally slower than local but scale at a similar rate.
+LOCAL_RESOURCES_DIR = (
+    Path(__file__).parent.parent / "aicsimageio" / "tests" / "resources"
```

Comment:
Possible change:
```suggestion
    Path(__file__).parents[1] / "aicsimageio" / "tests" / "resources"
```

Reply:
Ehhh. I like the `.parent.parent` personally. Something about a "reverse list of parents" weirds me out. If it was something like `Path(__file__).parents[-2]` where the parents are ordered "naturally" I would prefer haha